core/src/main/kotlin/org/evomaster/core/output/TestSuiteOrganizer.kt

```diff
@@ -228,8 +227,7 @@ class SortingHelper {
     /**
      *Sorting is done according to the comparator list. If no list is provided, individuals are sorted by max status.
      */
-    private fun sortByComparatorList (namingHelper: NamingHelper,
-                              comparators: List<Comparator<EvaluatedIndividual<*>>> = listOf(statusCode),
+    private fun sortByComparatorList (comparators: List<Comparator<EvaluatedIndividual<*>>> = listOf(statusCode),
```

Comment:
@arcuri82, I kept this just even though it just calls the naming strategy since there's quite an extensive comment explaining sorting and we're not modifying sorting yet. I'm guessing we'll get naming done first and then focus in the sorting bit